src/ess/dream/instrument_view.py

```diff
@@ -0,0 +1,148 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+from html import escape
+from typing import Any, Optional, Union
+
+import plopp as pp
+import scipp as sc
+from plopp.widgets import Box
```

Comment:
This now makes ipywidgets a hard dependency of the package, but I guess it's fine as we already have `pythreejs` as a dependency in the `pyproject.toml`

Reply:
Made the import conditional.